### PR TITLE
Parsed song attributes from song file.

### PR DESCRIPTION
**Pull request by pitchumani1989**
Modified .gitignore file to ignore ide files.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>regexp</groupId>
```
Comment (crunchersaspire) [inline pom.xml]: Do we need this dependency?

Comment (sskjames) [inline pom.xml]: Do we need this dependency?

File: src/test/java/org/worshipsongs/importer/SongParserTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    @Test
+    public void testParseSearchTitle()
+    {
+        assertEquals("foo@bar", parser.parseSearchTitle("Foo", "Bar"));
+        assertEquals("foo@", parser.parseSearchTitle("Foo", ""));
+        assertEquals("@", parser.parseSearchTitle("", ""));
+    }
+
+    @Test
+    public void testParseSearchLyrics()
```
Comment (sskjames) [inline src/test/java/org/worshipsongs/importer/SongParserTest.java:76]: This is not a song attribute.

File: pom.xml (hunk 1 above)
Comment (pitchumani1989) [inline pom.xml]: No, we don't need this dependency here. Can i remove it?

Comment (sskjames) [inline pom.xml]: Yes you can remove.

File: src/test/java/org/worshipsongs/importer/SongParserTest.java (hunk 2 above)
Comment (sskjames) [inline src/test/java/org/worshipsongs/importer/SongParserTest.java:76]: You can work on this later.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>worshipsongs-importer</groupId>
```
Comment (sskjames) [inline pom.xml]: Change groupId to "org.worshipsongs"

**Pull request merged**